pshtt.py

```diff
@@ -30,10 +37,10 @@
 # The fields we're collecting, will be keys in JSON and
 # column headers in CSV.
 HEADERS = [
-    "Domain", "Live", "Redirect",
+    "Domain", "Canonical URL", "Live", "Redirect",
     "Valid HTTPS", "Defaults HTTPS", "Downgrades HTTPS",
     "Strictly Forces HTTPS", "HTTPS Bad Chain", "HTTPS Bad Host Name",
-    "Expired Cert", "Weak Signature Chain", "HSTS", "HSTS Header",
+    "Expired Cert", "HSTS", "HSTS Header",
```

Comment:
I removed the `Weak Signature Chain` field, as I'd removed the code that produced it in a prior PR. Along with the `Canonical URL` field, this will mean some updated processing on DHS' side.